exercises/csharp/TriviaGame/TriviaGame/Game.cs

```diff
@@ -177,35 +199,45 @@ public bool WasCorrectlyAnswered()
             {
 
                 Console.WriteLine("Answer was corrent!!!!");
-                purses[currentPlayer]++;
-                Console.WriteLine(players[currentPlayer]
-                        + " now has "
-                        + purses[currentPlayer]
-                        + " Gold Coins.");
+                playerGoldCoins[_currentPlayer]++;
+                DeclareGoldCoins();
 
-                bool winner = DidPlayerWin();
-                currentPlayer++;
-                if (currentPlayer == players.Count) currentPlayer = 0;
+                bool winner = PlayerWon();
+                _currentPlayer++;
+                if (_currentPlayer == players.Count) _currentPlayer = 0;
 
                 return winner;
             }
         }
 
-        public bool WrongAnswer()
+        private void DeclareGoldCoins()
```

Comment:
I like that you've pulled this out into a separate function. Is Declare the best name for this? Maybe it is, I'm just wondering